resources/web/wwi/nodes/WbCadShape.js

```diff
@@ -50,18 +67,11 @@ export default class WbCadShape extends WbBaseNode {
 
     super.createWrenObjects();
 
-    if (this.urls.length === 0 || this.scene === 'undefined')
+    if (this.scene === 'undefined')
```

Comment:
Why remove the check on the urls length?

Reply:
sanity checks have been moved to the constructor, but I don't it it hurts to leave it it